example/android/app/build.gradle

```diff
@@ -210,13 +210,7 @@ dependencies {
         exclude group:'com.facebook.flipper'
     }
 
-    if (enableHermes) {
```

Comment:
there is a flag `enableHermes`, but it is not used after this change? maybe then it is worth to remove project property as well?